packages/logging-bunyan/test/index.js

```diff
@@ -37,6 +38,18 @@ describe('logging-bunyan', function() {
     };
   }
 
+  function fakeWritable(options) {
```

Comment:
`FakeWritable` (capitalization to indicate it's meant to be instantiated with `new`)